config/webpacker.yml

```diff
@@ -6,11 +6,11 @@ default: &default
   public_root_path: public
   public_output_path: packs
   cache_path: tmp/cache/webpacker
-  check_yarn_integrity: false
```

Comment:
Why was this omitted?

Reply:
![Uploading Screenshot 2020-10-07 at 12.40.04.png…]()

Reply:
<img width="1406" alt="Screenshot 2020-10-07 at 12 40 04" src="https://user-images.githubusercontent.com/286476/95326333-50e36e00-089a-11eb-8bdb-74d050ebd0f6.png">

Reply:
Sorry, should have mentioned that.

Reply:
Cool, thanks.